config/crd/bases/rsct.ibm.com_rscts.yaml

```diff
@@ -46,6 +46,12 @@ spec:
             type: object
           status:
             description: RSCTStatus defines the observed state of RSCT
+            properties:
+              current_status:
+                description: |-
+                  INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+                  Important: Run "make" to regenerate code after modifying this file
```

Comment:
Can you please update the `current_status`'s description ?

Reply:
sure